### PR TITLE
gui: send slot fee details

### PR DESCRIPTION
**Pull request by yufeng-jump**

### CODE REVIEW

File: src/disco/gui/fd_gui.c [hunk 1]
```diff
@@ -1247,8 +1248,9 @@ fd_gui_handle_completed_slot( fd_gui_t * gui,
   ulong failed_txn_count = msg[ 3 ];
   ulong nonvote_failed_txn_count = msg[ 4 ];
   ulong compute_units = msg[ 5 ];
-  ulong fees = msg[ 6 ];
-  ulong _parent_slot = msg[ 7 ];
+  ulong transaction_fee = msg[ 6 ];
+  ulong priority_fee = msg[ 7 ];
```
Comment (mmcgee-jump) [inline src/disco/gui/fd_gui.c:1252]: mcache MTU needs to be increased in the topology for this link

File: src/disco/gui/fd_gui_printf.c [hunk 2]
```diff
@@ -1002,8 +1002,15 @@ fd_gui_printf_slot( fd_gui_t * gui,
         else                                                 jsonp_ulong( gui, "failed_transactions", slot->failed_txn_cnt );
         if( FD_UNLIKELY( slot->compute_units==ULONG_MAX ) ) jsonp_null( gui, "compute_units" );
         else                                                jsonp_ulong( gui, "compute_units", slot->compute_units );
-        if( FD_UNLIKELY( slot->fees==ULONG_MAX ) ) jsonp_null( gui, "fees" );
-        else                                       jsonp_ulong( gui, "fees", slot->fees );
+        if( FD_UNLIKELY( slot->transaction_fee==ULONG_MAX || slot->priority_fee==ULONG_MAX ) ) {
+          jsonp_null( gui, "fees" );
```
Comment (mmcgee-jump) [inline src/disco/gui/fd_gui_printf.c:1006]: Don't need the "fees" field imo.

File: src/disco/gui/fd_gui_printf.c [hunk 3]
```diff
@@ -1002,8 +1002,15 @@ fd_gui_printf_slot( fd_gui_t * gui,
...
+          jsonp_null( gui, "transaction_fees" );
+          jsonp_null( gui, "priority_fees" );
+        } else {
+          jsonp_ulong( gui, "fees", slot->transaction_fee+slot->priority_fee ); // TODO Include JITO, rent?
+          jsonp_ulong( gui, "transaction_fees", slot->transaction_fee );
```
Comment (mmcgee-jump) [inline src/disco/gui/fd_gui_printf.c:1011]: Can you add these to the `websocket.md` documentation?

File: src/disco/gui/fd_gui.c (hunk 1 above)
Comment (yufeng-jump) [reply src/disco/gui/fd_gui.c:1252]: Good point.  It looks like the slot reset message is also sent over that link, and so it's been sized large enough to cover the chain of parents for a slot reset, and as a result also large enough for slot complete?

File: src/disco/gui/fd_gui_printf.c (hunk 3 above)
Comment (yufeng-jump) [reply src/disco/gui/fd_gui_printf.c:1011]: Oops.  Will do.

**Review by mmcgee-jump — CHANGES_REQUESTED** [review]
Can you rebase the agave changes into the existing replay stage GUI commit?

Also I'd add a comment to this part

```
/// FIREDANCER: GUI needs the full fee details to be filled in.
let _ = self.filter_program_errors_and_collect_fee_details(&execution_results);
```

File: src/disco/gui/fd_gui_printf.c (hunk 2 above)
Comment (mmcgee-jump) [reply src/disco/gui/fd_gui_printf.c:1006]: Can you check? If it does error we can just have allen remove it and package that with this same commit.

File: src/disco/gui/fd_gui.c (hunk 1 above)
Comment (mmcgee-jump) [reply src/disco/gui/fd_gui.c:1252]: Ah ok!

File: src/disco/gui/fd_gui_printf.c (hunk 2 above)
Comment (yufeng-jump) [reply src/disco/gui/fd_gui_printf.c:1006]: https://github.com/firedancer-io/firedancer-frontend/blob/b77f13657c72fc8408803a35413defb07502a9b5/src/api/entities.ts#L209

Maybe we just remove this line as part of this PR.

Comment (mmcgee-jump) [reply src/disco/gui/fd_gui_printf.c:1006]: Sure, if that fixes it feel free to push a commit to frontend

**Review by mmcgee-jump — APPROVED** [review]

**Review by mmcgee-jump — APPROVED** [review]

**Pull request merged**